### PR TITLE
Fix dataset and global tool bar error with standalone React component

### PR DESCRIPTION
**Pull request by vladimir-mck**
## Description

Resolves [#1351](https://github.com/kedro-org/kedro-viz/issues/1351)

## Development notes

- The Router is always rendered since the FlowChartWrapper components is using router hooks
- Dropdown useEffect condition is updated to run only if there is a prop 
- `layout` function is now async to fix `TypeError: getJob(...).then is not a function` error
## QA notes

**Steps to QA :**

1. create a new project using CRA, vite or some other tool 
2. install all the dependencies from kedro-viz package json into your new project
3. install `workerize-loader` from kedro-viz devDependencies
4. copy all the folders from src + config.js file into your project src folder
5. run `npm i`
6. import `App` from `src/compontents/App` into projects main component
7. run `npm start`

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [x] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1475"></a>

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -6,11 +6,12 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
-
 ## Bug fixes and other changes
 
 - Fix incorrect rendering of datasets in modular pipelines. (#1439)
 - Fix broken SVG/PNG exports in light theme. (#1463)
+- Fix error when setting globalToolbar prop value as false. (#1351)
+- Fix error error in dropdown component. (#1351)
```
Comment (ravi-kumar-pilla) [inline RELEASE.md:14]: can we have 1 release comment like **Fix dataset and global tool bar error with standalone React component** ?

**Review by ravi-kumar-pilla — CHANGES_REQUESTED** [review]

**Comment by ravi-kumar-pilla** [comment]
Hi @vladimir-mck ,

Nice work !! 

Can you please update the QA notes ? It will be helpful to quickly test the fix. Thank you !

File: RELEASE.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
-
 ## Bug fixes and other changes
 
 - Fix incorrect rendering of datasets in modular pipelines. (#1439)
 - Fix broken SVG/PNG exports in light theme. (#1463)
+- Fix dataset and global tool bar error with standalone React component (#1351)
```
Comment (tynandebold) [inline RELEASE.md:13]:
```suggestion
- Fix dataset and global toolbar error with standalone React component (#1351)
```

**Review by tynandebold — APPROVED** [review]
Looks good! Thanks for your first PR. Here's to many more 💥

**Comment by ravi-kumar-pilla** [comment]
Hi @vladimir-mck, I am trying to test your PR locally

But, I am struggling to follow your QA notes. Do you think if we can use - https://medium.com/dailyjs/how-to-use-npm-link-7375b6219557 to test quickly ?

When I am copying package.json from kedro viz and doing npm install in the new project created via CRA, I get lot of dependency errors. Is there a better way to test ? Thank you !

**Review by ravi-kumar-pilla — APPROVED** [review]
LGTM

**Pull request merged**